<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AUserInterfaceClass</class>
 <widget class="QWidget" name="AUserInterfaceClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AUserInterface</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="TopLayout">
     <item>
      <widget class="QGroupBox" name="VisibleGroup">
       <property name="title">
        <string>Visible</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="VisibleDefault">
          <property name="text">
           <string>Default</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="VisibleHBoard">
          <property name="text">
           <string>HBoard</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="VisibleVBoard">
          <property name="text">
           <string>VBoard</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="VisibleLineLaser">
          <property name="text">
           <string>Line Laser</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="VisibleCursorPosition">
          <property name="text">
           <string>Cursor Position</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="UpdateGroup">
       <property name="title">
        <string>Update</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QCheckBox" name="UpdateHBoardBox">
          <property name="text">
           <string>Update H Board</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="UpdateVBoardBox">
          <property name="text">
           <string>Update V Board</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="UpdateLineLaserBox">
          <property name="text">
           <string>Update Line Laser</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="UpdateNoneBox">
          <property name="text">
           <string>Update None</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Information</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="InfoHBoard">
          <property name="text">
           <string>H Board</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="InfoVBoard">
          <property name="text">
           <string>V Board</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="InfoLineLaser">
          <property name="text">
           <string>Line Laser</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="BotLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Scan</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="ScanBox">
          <property name="text">
           <string>Scan</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ExportBox">
          <property name="text">
           <string>Export</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="AUserInterface.qrc"/>
 </resources>
 <connections/>
</ui>
